### ISSUE TITLE
如何修改单卡策略（学习率、批量大小等）

### ISSUE THREAD

**Issue by monkeycc** [PaddlePaddle/PaddleClas#2532]
The training strategy provided by PaddleClas is based on 4 gpus. But the number of gpu is 1 in current training. Please modify the stategy (learning rate, batch size and so on) if use this config to train.
PaddleClas提供的训练策略基于4 gpus。但在目前的训练中，gpu的数量是1。如果使用此配置进行训练，请修改策略（学习率、批量大小等）。

如何修改单卡策略（学习率、批量大小等）

是否可以考虑出一个单卡转换脚本

比如选择 3060 单卡  选择模型
就会出 学习率、批量大小 的结果

这样是不是更加人性化呢

**Comment by cuicheng01** [comment]
如果是单卡的话，学习率除以4就可以～

**Comment by happybear1015** [comment]
> 如果是单卡的话，学习率除以4就可以～

那batch_size呢？
